test/VestingManagementTest.ts

```diff
@@ -0,0 +1,390 @@
+import { ethers } from "hardhat";
+import chai, { expect } from "chai";
+import { FakeContract } from "@defi-wonderland/smock/dist/src/types";
+import { smock } from "@defi-wonderland/smock";
+import { SignerWithAddress } from "@nomiclabs/hardhat-ethers/signers";
+import { VestingManagement } from "../typechain";
+import { deployContract } from "../utils";
+import IERC20 from "../artifacts/@openzeppelin/contracts/token/ERC20/IERC20.sol/IERC20.json";
+
+const { time, expectRevert } = require("@openzeppelin/test-helpers");
+
+chai.use(smock.matchers);
+
+describe("VestingManagement", () => {
+  let vestingManagement: VestingManagement;
+  let BENEFICIARY: string;
+  let START_TIME: number;
+  let DURATION: number;
+  let vestedToken: FakeContract;
+  let signers: SignerWithAddress[];
+
+  beforeEach(async () => {
+    signers = await ethers.getSigners();
+    vestedToken = await smock.fake(IERC20.abi);
+    BENEFICIARY = signers[9].address;
+    const FIRST_MINED_TIMESTAMP: number = await time.latest();
+    START_TIME = +FIRST_MINED_TIMESTAMP + 100;
```

Comment:
why `+` sign here before `FIRST_MINED_TIMESTAMP`?

Reply:
so that `START_TIME` is not treated as string (tests fail because string concatenation takes place instead of integer addition)

Reply:
But it is declared as `number`